packages/cli/src/commands/setup/deploy/providers/aws-serverless.js

```diff
@@ -24,7 +25,7 @@ custom:
 
 provider:
   name: aws
-  runtime: nodejs12.x
+  runtime: nodejs14.x
```

Comment:
Ah, very nice. FYI: our build targets Nodejs 12, which shouldn't be a problem. But good to see 14 is available 'cause I'd like us to bump our internal transpile target in the future. (Not now, but likely once we approach Redwood v1 GA.

Reply:
Sound good, I will remove this change from the PR; is there another ticket where we might want to track the upgrade to 14?

Reply:
Ah, I wasn't implying you need to remove it! Let's leave it for now but definitely confirm/review before finalizing this. 

I just meant to say:
1. it's good to see that AWS Lambdas has v14 support (it does, correct?)
2. this will help us force the conversation about if/when we bump our targets. And we also might need to document how the transpiration target can be set per-project's needs.

Reply:
You can check available runtimes [here](https://docs.aws.amazon.com/lambda/latest/dg/lambda-nodejs.html), but yes, Node 14 has been supported on [Lambda since February](https://aws.amazon.com/blogs/compute/node-js-14-x-runtime-now-available-in-aws-lambda/) and on [Lambda@Edge since April](https://aws.amazon.com/about-aws/whats-new/2021/04/lambda-edge-support-node14/). There's an [open issue for Node 16](https://github.com/aws/aws-lambda-base-images/issues/14) but wouldn't count on that coming out anytime soon.